.vscode/launch.json

```diff
@@ -7,7 +7,8 @@
       "port": 9003,
       "pathMappings": {
         "/home/site/wwwroot/": "${workspaceFolder}"
-      }
+      },
+      "hostname": "localhost"
```

Comment:
Hmm. It doesn't work for me without hostname defined like this. For the record, I'm running in WSL on Windows, everything in Docker, and debugging a process I start with `docker-compose exec -w /home/site/wwwroot/api webserver php artisan test`. What's your setup?

We could specify OS-specific fields [like so](https://code.visualstudio.com/docs/python/environments#_environment-variable-definitions-file), but I suspect our discrepancy has more to do with docker than our OS 😅. 

... But replacing this change with
```
      "linux": {
        "hostname": "localhost",
      },
```
works for me, does that work for your setup?

Reply:
Yeah, that's still broken for me.  Since I'm on native Linux that config change is still effective for me.  Really, we need a "wsl" environment for configs but I don't think that exists.  For me, xdebug needs to find the debugger in the Docker host machine while on WSL it needs needs to find the debugger in a peer container.  Maybe this will clear up once we start working with `hosts` file overrides.  In the meantime, go ahead and merge this.  I'm the only Linux user on the team who isn't using WSL so I'll work around it until I can figure out a fix.

Reply:
Okay, I'll leave it in but add a comment about this.